app/scripts/modules/core/src/managed/ArtifactDetail.tsx

```diff
@@ -51,105 +53,126 @@ export const ArtifactDetail = ({
           {/* a short summary with actions/buttons will live here */}
           <div className="detail-section-right">{/* artifact metadata will live here */}</div>
         </div>
-        {environments.map(({ name: environmentName, state, deployedAt, replacedAt, replacedBy }) => {
-          const deployedAtMillis = DateTime.fromISO(deployedAt).toMillis();
-          const replacedAtMillis = DateTime.fromISO(replacedAt).toMillis();
-          const { version: replacedByPackageVersion, buildNumber: replacedByBuildNumber } =
-            parseName(replacedBy || '') || {};
+        {environments.map(
+          ({ name: environmentName, state, deployedAt, replacedAt, replacedBy, statefulConstraints }) => {
```

Comment:
prettier totally re-indented this whole function because of this extra restructuring, so the diff is ridiculous and unreadable. I recommend turning on the "Hide whitespace changes" option in the diff settings menu as it cleans up nicely.